code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -94,26 +94,27 @@
 				new cat_type(loc)
 
 
-/mob/living/simple_animal/pet/cat/handle_automated_action()
+/mob/living/simple_animal/pet/cat/Life()
 	..()
-	if(prob(1))
-		custom_emote(1, pick("stretches out for a belly rub.", "wags its tail.", "lies down."))
-		icon_state = "[icon_living]_rest"
-		resting = 1
-		update_canmove()
-	else if (prob(1))
-		custom_emote(1, pick("sits down.", "crouches on its hind legs.", "looks alert."))
-		icon_state = "[icon_living]_sit"
-		resting = 1
-		update_canmove()
-	else if (prob(1))
-		if (resting)
-			custom_emote(1, pick("gets up and meows.", "walks around.", "stops resting."))
-			icon_state = "[icon_living]"
-			resting = 0
+	make_babies()
+
+
+/mob/living/simple_animal/pet/cat/handle_automated_action()

```

Comment:
My understanding is that handle_automated_action() only runs if the AI is in state AI_ON. 
I thought pets never entered AI_ON because they never aggro on targets, as they're not hostile mobs?

Reply:
the default state of simple mob's AI status is `AI_ON`.